Sohail-sohailkamdar/Homework-1/BONUS-madlibs/js/app.js

```diff
@@ -0,0 +1,26 @@
+
+function madlibs(){
+    let startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+    let startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+    let random1 = Math.floor((Math.random() * startupX.length));
+    let random2 = Math.floor((Math.random() * startupY.length));
+    var txtNode = 'A startup that is ' + startupX[random1] + ', but for ' + startupY[random2];
+    document.getElementById('xForY').textContent = txtNode;
+}
+
+var favorites=[];
```

Comment:
The new version of JavaScript, ES6, introduces let in favor of var; this new type of variable keeps scope, which works great in loops and conditionals.

You also want to keep it consistent across all your declarations if you can.